bin/upload-staticfiles.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -exo pipefail
+
+source docker/bin/set_git_env_vars.sh
+
+if [[ "$BRANCH_NAME" != "prod" ]]; then
+    # nothing to do if not prod deployment
+    exit 0
+fi
+
+CONTAINER_NAME="bedrock-${BRANCH_AND_COMMIT}"
+
+rm -rf static
+# have to rerun staticfiles without symlinks or we just copy broken symlinks
+docker-compose run --name "$CONTAINER_NAME" release docker/bin/build_staticfiles.sh --nolink
+docker cp "${CONTAINER_NAME}:/app/static" ./static
+docker rm -f "$CONTAINER_NAME"
+
+cd static
+for BUCKET in stage prod; do
+    # hashed filenames
+    aws s3 sync \
+        --exclude "*" \
+        --include "*.????????????.*" \
+        --acl public-read \
+        --cache-control "max-age=315360000, public, immutable" \
+        --profile bedrock-media \
+        . "s3://bedrock-${BUCKET}-media/"
+    # non-hashed filenames
```

Comment:
are lines 30 and 31 the same comment repeated?

Reply:
They look different to me locally. Line 30 says what the next section is doing, and 31 says that we may not eventually need to do this and 32 says we should look into that someday if it's a problem.